app_flutter/test/service/appengine_cocoon_test.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:app_flutter/service/appengine_cocoon.dart';
+import 'package:cocoon_service/protos.dart'
+    show Commit, CommitStatus, Stage, Task;
+import 'package:test/test.dart';
+
+import 'package:http/http.dart' show Response;
+import 'package:http/testing.dart';
+import 'package:fixnum/fixnum.dart';
+
+void main() {
+  // This is based off data the Cocoon backend sends out from v1.
+  // It doesn't map directly to protos since the backend does
+  // not use protos yet.
+  final String jsonGetStatsResponse = """
```

Comment:
I would move this up to a top level const in the file

Reply:
Done